pkg/providers/aws/cluster_network_provider_test.go

```diff
@@ -1262,6 +1376,11 @@ func TestNetworkProvider_ReconcileNetworkProviderConfig(t *testing.T) {
 							}),
 						}}, nil
 					}
+					ec2Client.describeSubnetsFn = func(input *ec2.DescribeSubnetsInput) (*ec2.DescribeSubnetsOutput, error) {
+						return &ec2.DescribeSubnetsOutput{
+							Subnets: buildValidBundleSubnets(),
```

Comment:
this too, as we are trying to verify the cidr block, we can not be using a valid bundled subnet

Reply:
replaced with standalone